### ISSUE TITLE
Admin link control is only configurable via system setting

### ISSUE THREAD

**Issue by christophertubbs** [rq/django-rq#495]
Currently, the only way to show or hide the admin link is to set `RQ_SHOW_ADMIN_LINK = True` in the system settings for your django server. Unfortunately, unless there are some knobs and dials I haven't found, that only shows it for **all** users that can access the admin site or absolutely none of them. Setting a link directly to the queues is workable at the very least, which is nice. That still presents a situation where users that I _don't_ want to have access to the queue now have access to the queue.

What would be preferable would be access to be controlled via permissions so I can easily dictate who has access to what resources on the admin site like just about any other app.

**Comment by selwin** [comment]
Yes, having a permission to view Django-RQ pages would be nice. A PR for this is welcome :)

**Comment by haakenlid** [comment]
#512 adds custom permissions to the admin views
